pvlib/tracking.py

```diff
@@ -247,13 +257,13 @@ def __repr__(self):
 
 def singleaxis(apparent_zenith, apparent_azimuth,
                axis_tilt=0, axis_azimuth=0, max_angle=90,
-               backtrack=True, gcr=2.0/7.0):
+               backtrack=True, gcr=2.0/7.0, side_slope=0):
     """
     Determine the rotation angle of a single axis tracker when given a
-    particular sun zenith and azimuth angle. See [1]_ for details about
-    the equations.
-    Backtracking may be specified, and if so, a ground coverage
-    ratio is required.
+    particular sun zenith and azimuth angle.
+
+    See [1]_, [2]_ for details about the equations. Backtracking may be
```

Comment:
Is it necessary to retain [1], or does [2] support all the calculations done here? The two papers use different methods, citing both may make it more difficult for someone who is trying to follow the calculation.